src/Storages/System/InformationSchema/tables.sql

```diff
@@ -3,15 +3,26 @@ ATTACH VIEW tables
     `table_catalog` String,
     `table_schema` String,
     `table_name` String,
-    `table_type` Enum8('BASE TABLE' = 1, 'VIEW' = 2, 'FOREIGN TABLE' = 3, 'LOCAL TEMPORARY' = 4, 'SYSTEM VIEW' = 5),
+    `table_type` String,
+    `table_comment` Nullable(String),
+    `table_collation` Nullable(String),
     `TABLE_CATALOG` String ALIAS table_catalog,
     `TABLE_SCHEMA` String ALIAS table_schema,
     `TABLE_NAME` String ALIAS table_name,
-    `TABLE_TYPE` Enum8('BASE TABLE' = 1, 'VIEW' = 2, 'FOREIGN TABLE' = 3, 'LOCAL TEMPORARY' = 4, 'SYSTEM VIEW' = 5) ALIAS table_type
+    `TABLE_TYPE` String ALIAS table_type,
+    `TABLE_COMMENT` Nullable(String) ALIAS table_comment,
+    `TABLE_COLLATION` Nullable(String) ALIAS table_collation
 ) AS
 SELECT
-    database AS table_catalog,
-    database AS table_schema,
-    name AS table_name,
-    multiIf(is_temporary, 4, engine like '%View', 2, engine LIKE 'System%', 5, has_own_data = 0, 3, 1) AS table_type
+    database  AS table_catalog,
+    database  AS table_schema,
+    name      AS table_name,
+    comment   AS table_comment,
+    multiIf(is_temporary,          'LOCAL TEMPORARY', 
+            engine LIKE '%View',   'VIEW', 
+            engine LIKE 'System%', 'SYSTEM VIEW', 
+            has_own_data = 0,      'FOREIGN TABLE', 
+            'BASE TABLE'
+            ) AS table_type,
+    'utf8mb4' AS table_collation
```

Comment:
1. MySQL [shows](https://onecompiler.com/mysql/3zmzeeb2j) `utf8mb4_0900_ai_ci`. How is that different to `utf8mb4`.

2. Why are fields 'table_collation` and `table_comment` of type `Nullable`? As far as I see, there is no situation where they will be `NULL`.

Reply:
https://onecompiler.com/mysql/3zmzs9sdz

```sql
DESCRIBE information_schema.tables;
```

shows

```
Field	        Type	        Null	        Key	Default	   Extra
...
TABLE_COLLATION	varchar(64)	YES		NULL	
TABLE_COMMENT	text	        YES		NULL	
```

Reply:
Regarding `utf8mb4` and `utf8mb4_0900_ai_ci` - I am confused now :) See your comment here: https://github.com/ClickHouse/ClickHouse/pull/54115#discussion_r1317076889

> Hardcoding the table collation as utf8mb4_0900_ai_ci is weird. 1. ClickHouse has no per-column/per-table/per-schema collations. We can surely hardcode something Unicode-like but then let's display something more generic, e.g. just utf8 - assuming Tableau Online does not expect exactly utf8mb4_0900_ai_ci.